PHPCI/Plugin/Email.php

```diff
@@ -70,12 +70,21 @@ public function execute()
 
         $buildStatus  = $this->build->isSuccessful() ? "Passing Build" : "Failing Build";
         $projectName  = $this->build->getProject()->getTitle();
-        $mailTemplate = $this->build->isSuccessful() ? 'Email/success' : 'Email/failed';
+        $mailTemplate = $this->build->isSuccessful() ? 'Email/long' : 'Email/short';
+
+        if (!empty($this->options['template'])) {
```

Comment:
Is it possible to check if the template exist? (If I made a typo it can result in an error)